src/instance.js

```diff
@@ -118,6 +129,101 @@ Instance.prototype.create = function(options, callback) {
   this.bigtable.createInstance(this.name, options, callback);
 };
 
+/**
+ * Create an app profile.
+ *
+ * @param {string} name The name to be used when referring to the new
+ *     app profile within its instance.
+ * @param {object} options AppProfile creation options.
+ * @param {'any'|Cluster} options.routing  The routing policy for all
```

Comment:
If this type is `Cluster` then `options.routing,name` is required.  I think that's important for the docs.  It might also be worth while using `options.routing.clusterId` instead of `name` so that the usage is clearer.

Reply:
oh.  I see.  This needs to be an instance of a `Cluster` object.  That's confusing.  Would `any|[a clusterId in the instance]` work  instead of a `Cluster` object?

Reply:
I believe the jsdoc system expects a format the way it is. If the user provides a cluster not in the instance, the API will reject the call